pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+# Pull Request Template
+
+#issue
+
+## Description
+
+Please include a summary of the change and which issue(s) is addressed. Please also include relevant motivation and context.
+
+## How Can This Be Tested/Reviewed?
+
+Provide instructions so we can review.
+
+Describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.
+
+## Checklist:
+
+- [ ] My code follows the style guidelines of this project
```

Comment:
are the style guidelines the style conventions in the README?

Reply:
yep! i also have an incoming pr to update the readme with some more detail about picking up issues / contributing